src/molecules/product-list-item/ProductListItem.js

```diff
@@ -0,0 +1,95 @@
+// @vue/component
+export default {
+  model: {
+    prop: 'qty',
+    event: 'change'
+  },
+  props: {
+    /**
+     * Product id
+     */
+    id: {
+      type: String,
+      required: true
+    },
+    /**
+     * Product name
+     */
+    name: {
+      type: String,
+      required: true
+    },
+    /**
+     * Product url
+     */
+    url: {
+      type: String,
+      required: true
+    },
+    /**
+     * Product image url
+     */
+    imageUrl: {
+      type: String,
+      required: true
+    },
+    /**
+     * Product standard price
+     */
+    price: {
+      type: String,
+      required: true
+    },
+    /**
+     * Product special price
+     */
+    specialPrice: {
+      type: String,
+      required: true
+    },
+    /**
+     * Product old price
+     */
+    oldPrice: {
+      type: String,
+      required: true
+    },
```

Comment:
IMO it would be better to pass one object price as storefront did here https://github.com/DivanteLtd/storefront-ui/blob/master/packages/vue/src/components/molecules/SfProductCard/SfProductCard.html